lib/github_changelog_generator/generator/generator_generation.rb

```diff
@@ -21,7 +21,8 @@ def compound_changelog
 
       log += File.read(options[:base]) if File.file?(options[:base])
 
-      log += "\n\n\\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*"
+      credit_line = "\* *This Change Log was automatically generated by [github_changelog_generator](https://github.com/skywinder/Github-Changelog-Generator)*"
+      log += "\n\n\#{credit_line}" if log.include? credit_line == false
```

Comment:
@olleolleolle do you know where the extra '\'? Should it be `"\n\n#{credit_line}"`?